client/my-sites/domains/domain-management/transfer/transfer-to-other-site/index.jsx

```diff
@@ -115,52 +117,84 @@ export class TransferToOtherSite extends React.Component {
 		}
 	};
 
-	render() {
-		if ( ! this.isDataReady() ) {
-			return <DomainMainPlaceholder goBack={ this.goToEdit } />;
+	getMessage() {
+		const {
+			selectedDomainName: domainName,
+			domainsWithPlansOnly,
+			isMappedDomain,
+			translate,
+		} = this.props;
+		const translateArgs = { args: { domainName }, components: { strong: <strong /> } };
+		if ( isMappedDomain ) {
+			if ( domainsWithPlansOnly ) {
+				return translate(
+					'Please choose a site with a paid plan ' +
+						"you're an administrator on to transfer mapping of {{strong}}%(domainName)s{{/strong}} to:",
```

Comment:
The language here doesn't quite feel right/clear to me, but I don't have a great alternative. I am thinking it may be better to phrase this as:
`you're an administrator on to transfer the domain mapping for {{strong}}%(domainName)s{{/strong}} to:`
This applies here and below.

Reply:
Thanks, that's indeed clearer 👍